accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -6,6 +6,11 @@ Status: Draft
 
 ## CHANGELOG
 
+2020..06.02
```

Comment:
FYI: double dots here.

Reply:
Fixed in 8c3bf5e2ba6b6f38dbf66fd7a39e2f84a3463eba.